lib/gooddata/lcm/actions/associate_clients.rb

```diff
@@ -46,7 +49,21 @@ class AssociateClients < BaseAction
       ]
 
       class << self
+        DELETE_MODES = %w(
+          none
+          delete_projects
+          delete_extra
+        )
         def call(params)
+          if params.delete_projects && !params.delete_extra
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)